libs/sdk-ui-ext/src/internal/components/pluggableVisualizations/areaChart/PluggableAreaChart.tsx

```diff
@@ -194,26 +195,29 @@ export class PluggableAreaChart extends PluggableBaseChart {
     }
 
     private getAllAttributesWithoutDate(buckets: IBucketOfFun[]): IBucketItem[] {
-        return this.getAllAttributes(buckets).filter(negate(isDateBucketItem));
+        return this.getAllAttributes(buckets).filter(isNotDateBucketItem);
     }
 
     private filterStackItems(bucketItems: IBucketItem[]): IBucketItem[] {
-        return bucketItems.filter(negate(isDateBucketItem)).slice(0, MAX_STACKS_COUNT);
+        return bucketItems.filter(isNotDateBucketItem).slice(0, MAX_STACKS_COUNT);
     }
 
     private getBucketItems(referencePoint: IReferencePoint) {
         const buckets = get(referencePoint, BUCKETS, []);
         const measures = getFilteredMeasuresForStackedCharts(buckets);
         const dateItems = getDateItems(buckets);
+        const firstDateItem = dateItems[0];
```

Comment:
It would be maybe worthy to put here some comments about why we pick the first date.